app/screens/snack_bar/index.tsx

```diff
@@ -115,7 +117,7 @@ const SnackBar = ({barType, componentId, onAction, sourceScreen}: SnackBarProps)
                     width: (SNACK_BAR_WIDTH / 100) * diffWidth,
                 };
                 break;
-            case [Screens.PERMALINK, Screens.MENTIONS, Screens.SAVED_MESSAGES].includes(sourceScreen):
+            case ([Screens.PERMALINK, Screens.MENTIONS, Screens.SAVED_MESSAGES] as AvailableScreens[]).includes(sourceScreen):
```

Comment:
0/5 this line can be less cluttered if we define this (maybe outside of the component) in a constant.
i.e.
```
const caseScreens: AvailableScreens[] = [Screens.PERMALINK, Screens.MENTIONS, Screens.SAVED_MESSAGES];
...
case caseScreens.includes(sourceScreen):
```
(of course, the name of the constant relevant to this context)